test/test_datetimeWrapper.js

```diff
@@ -602,8 +602,8 @@ describe('sundial', function() {
         expect(Object.keys(tzs)).to.deep.equal(keys);
         expect(tzs.bigFour.length).to.equal(4);
         expect(tzs.unitedStates.length).to.equal(9);
-        expect(tzs.hoisted.length).to.equal(95);
-        expect(tzs.theRest.length).to.equal(441);
+        expect(tzs.hoisted.length).to.equal(98);
```

Comment:
What are these numbers?

Reply:
Don't laugh.
The world is basically split in 3:
- US timezones (big4)
- English speaking ones (hoisted)
- the rest
As we upgraded moment timezones, some new timezones are falling in the 2 last groups 😁

Reply:
:+1: